test/microphysics2M_tests.jl

```diff
@@ -577,20 +578,24 @@ function test_microphysics2M(FT)
             TT.@test DT.exponential_cdf(Dr_mean, D_max) ≈ 1 - p
 
             # Sanity checks for number concentrations for rain
-            ND = QGK.quadgk(f_D, D_min, D_max)[1]
-            Nx = QGK.quadgk(f_x, x_min, x_max)[1]
-            ND_psd = QGK.quadgk(psd, D_min, D_max)[1]
-            TT.@test ND ≈ Nᵣ rtol = 1e-5
-            TT.@test Nx ≈ Nᵣ rtol = 5e-3
-            TT.@test ND_psd ≈ Nᵣ rtol = 1e-5
+            ND = P3.integrate(f_D, D_min, D_max; quad = P3.ChebyshevGauss(1000))
+            Nx = P3.integrate(f_x, x_min, x_max; quad = P3.ChebyshevGauss(100_000))
```

Comment:
Is that a realistic number of points for actual simulations? Shoul we be testing with such high numbers?

Reply:
No, it's way too many for practical simulations. I don't yet know why I needed so many points to get reasonable accuracy. but could try to investigate. I suspect it has to do with the physical quantities/units involved in the integral. In other words, rescaling/non-dimensionalizing the integrands could lead to better performance.

In any case, we don't actually ever integrate the distribution over mass, so it doesn't matter too much. The key point is that we should get the same number integrating over mass or diameter.